op-defender/psp_executor/README.md

```diff
@@ -8,28 +8,75 @@ The service is designed to listen on a port and execute a PSP onchain when a req
 
 ## 1. Usage
 
-### 1 .Run HTTP API service
+### 1. Run HTTP API service
 
 To start the HTTP API service we can use the following oneliner command:
-![f112841bad84c59ea3ed1ca380740f5694f553de8755b96b1a40ece4d1c26f81](https://github.com/user-attachments/assets/17235e99-bf25-40a5-af2c-a0d9990c6276)
+<img width="3357" alt="image" src="https://github.com/user-attachments/assets/8029c347-7dbb-4926-8e69-c4737b2d9f17">
+
 Settings of the HTTP API service:
 
-| Port                          | API Path             | HTTP Method |
-| ----------------------------- | -------------------- | ----------- |
-| 8080 (Default can be changed) | `/api/psp_execution` | POST        |
+| Port           | API Path             | HTTP Method |
+| -------------- | -------------------- | ----------- |
+| 8080 (Default) | `/api/psp_execution` | POST        |
+| 8080 (Default) | `/api/healtcheck`    | GET         |
+
+To run the psp_executor service, you can use the following command:
 
 ```shell
-go run ../cmd/defender psp_executor --privatekey XXXXXX --receiver.address 0xDEADBEEFDEADBEEFDEADBEEFDEADBEEFDEADBEEF --rpc.url http://localhost:8545 --port.api 8080
+go run ../cmd/defender psp_executor --privatekey 2a[..]c6 --safe.address 0x837DE453AD5F21E89771e3c06239d8236c0EFd5E --path /tmp/psps.json --chainid 11155111 --superchainconfig.address 0xC2Be75506d5724086DEB7245bd260Cc9753911Be --rpc.url http://localhost:8545 --port.api 8080
+```
+
+Explanation of the options:
+| Argument | Value | Explanation |
+| ---------------------------- | ------------------------------------------ | ------------------------------------ |
+| `--privatekey` | 2a[..]c6 | Private key for transaction signing |
+| `--safe.address` | 0x837DE453AD5F21E89771e3c06239d8236c0EFd5E | Address of the Safe contract that presigned the transation|
+| `--path` | /tmp/psps.json | Path to JSON file containing PSPs |
+| `--chainid` | 11155111 | Chain ID for the network |
+| `--superchainconfig.address` | 0xC2Be75506d5724086DEB7245bd260Cc9753911Be | Address of SuperchainConfig contract |
+| `--rpc.url` | http://localhost:8545 | URL of the RPC node |
+| `--port.api` | 8080 | Port for the HTTP API server |
+
+**PSPs Format**
+The PSPs are stored with a JSON format. The JSON file should contain an array of PSPs. Each PSP should have the following fields:
+
+```JSON
```

Comment:
Maybe yaml would be better for configuration for readability and the option to add comments?
Not a strong opinion, just adding as possibility.

Reply:
Thanks for the suggestion!
 Actually this is the output we have from superchains-ops (this is JSON).
-> https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/sep/017-presigned-pause
For me, this is not good to reparse to YAML from to JSON each time (if the fields are changing etc).